spec/support/request_specs.rb

```diff
@@ -1,11 +1,24 @@
 require_relative(File.expand_path('../../../app/service', __FILE__))
```

Comment:
did you already change this? I would also only use `require_relative '../file'`. I can merge it into master then :-)